src/components/new-safe/CreateSafeInfos/index.tsx

```diff
@@ -0,0 +1,42 @@
+import InfoWidget from '@/components/create-safe/InfoWidget'
+import { Grid } from '@mui/material'
+import type { AlertColor } from '@mui/material'
+import { type ReactElement } from 'react'
+
+export type CreateSafeInfoItem = {
+  title: string
+  variant: AlertColor
+  steps: { title: string; text: string | ReactElement }[]
+}
+
+const CreateSafeInfos = ({
+  staticHint,
+  dynamicHint,
+}: {
+  staticHint?: CreateSafeInfoItem
+  dynamicHint?: CreateSafeInfoItem
+}) => {
+  return (
```

Comment:
Is there ever the case of having no hints? We could `return null` if so.

Reply:
added that return